### PR TITLE
update UCSD cache info; #STAS-54

### PR DESCRIPTION
**Pull request by zvada**

### CODE REVIEW

**Comment by zvada** [comment]
@DrDaveD, please review when you can.

File: goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ]; then
 else
     CVMFS_SERVER_URL="http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1goc.opensciencegrid.org/cvmfs/@fqrn@"
 fi
-CVMFS_EXTERNAL_URL="http://osgxroot.usatlas.bnl.gov:8000/;http://xrd-cache-1.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/;http://fiona.uvalight.net:8000/;http://osg-chicago-stashcache.nrp.internet2.edu:8000/;http://osg-new-york-stashcache.nrp.internet2.edu:8000/;http://sc-cache.chtc.wisc.edu:8000/;http://osg-gftp.pace.gatech.edu:8000/"
+CVMFS_EXTERNAL_URL="http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/;http://fiona.uvalight.net:8000/;http://osg-chicago-stashcache.nrp.internet2.edu:8000/;http://osg-new-york-stashcache.nrp.internet2.edu:8000/;http://sc-cache.chtc.wisc.edu:8000/;http://osg-gftp.pace.gatech.edu:8000/"
```
Comment (DrDaveD) [inline goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf:7]: This service does not work.  debug log shows
```
(download) escaped http://stashcache.t2.ucsd.edu:8000//pnfs/fnal.gov/usr/nova/da
ta/flux/ppfx/flux_TA_ND_FHC_fid_coh.root to http://stashcache.t2.ucsd.edu:8000//
pnfs/fnal.gov/usr/nova/data/flux/ppfx/flux_TA_ND_FHC_fid_coh.root    [03-14-2019
 11:48:49 CDT]
(download) http redirect: HTTP/1.1 302 Redirect
    [03-14-2019 11:48:50 CDT]
(download) Location: http://stashcache.t2.ucsd.edu:9001/pnfs/fnal.gov/usr/nova/d
ata/flux/ppfx/flux_TA_ND_FHC_fid_coh.root
    [03-14-2019 11:48:50 CDT]
(download) Verify downloaded url /pnfs/fnal.gov/usr/nova/data/flux/ppfx/flux_TA_ND_FHC_fid_coh.root, proxy DIRECT (curl error 28)    [03-14-2019 11:49:20 CDT]
(download) Trying again on same curl handle, same url: 1, error code 15    [03-14-2019 11:49:20 CDT]
(download) backing off for 1093 ms    [03-14-2019 11:49:20 CDT]
(download) http redirect: HTTP/1.1 302 Redirect
    [03-14-2019 11:49:22 CDT]
(download) Location: http://stashcache.t2.ucsd.edu:9001/pnfs/fnal.gov/usr/nova/data/flux/ppfx/flux_TA_ND_FHC_fid_coh.root
    [03-14-2019 11:49:22 CDT]
(download) http status error code: HTTP/1.1 404 Not Found
```

File: goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CVMFS_EXTERNAL_URL="https://xrootd-local.unl.edu:1094//user/ligo/;https://its-condor-xrootd1.syr.edu:8443//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;http://osg-gftp.pace.gatech.edu:8443//user/ligo/"
+CVMFS_EXTERNAL_URL="https://xrootd-local.unl.edu:1094//user/ligo/;https://stashcache.t2.ucsd.edu:8444//user/ligo/;https://its-condor-xrootd1.syr.edu:8443//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;http://osg-gftp.pace.gatech.edu:8443//user/ligo/"
```
Comment (DrDaveD) [inline goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf]: This new service works for me.

File: goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf [hunk 3]
```diff
@@ -4,7 +4,7 @@ if [ "$CVMFS_HTTP_PROXY" = "DIRECT" ]; then
 else
     CVMFS_SERVER_URL="http://cvmfs-s1bnl.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1fnal.opensciencegrid.org:8000/cvmfs/@fqrn@;http://cvmfs-s1goc.opensciencegrid.org/cvmfs/@fqrn@"
 fi
-CVMFS_EXTERNAL_URL="http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/;http://fiona.uvalight.net:8000/;http://osg-chicago-stashcache.nrp.internet2.edu:8000/;http://osg-new-york-stashcache.nrp.internet2.edu:8000/;http://sc-cache.chtc.wisc.edu:8000/;http://osg-gftp.pace.gatech.edu:8000/"
+CVMFS_EXTERNAL_URL="http://osgxroot.usatlas.bnl.gov:8000/;http://stashcache.t2.ucsd.edu:8000/;http://mwt2-stashcache.campuscluster.illinois.edu:8000/;http://its-condor-xrootd1.syr.edu:8000/;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/;http://fiona.uvalight.net:8000/;http://osg-chicago-stashcache.nrp.internet2.edu:8000/;http://osg-new-york-stashcache.nrp.internet2.edu:8000/;http://sc-cache.chtc.wisc.edu:8000/;https://osg-gftp.pace.gatech.edu:8000/"
```
Comment (DrDaveD) [inline goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf]: No, we don't want https for any repositories but ligo.

Comment (zvada) [reply goc/config-osg/etc/cvmfs/domain.d/osgstorage.org.conf]: good catch, thanks!

**Review by DrDaveD — APPROVED** [review]
Looks good

**Pull request merged**